security/maltrail/src/opnsense/mvc/app/controllers/OPNsense/Maltrail/forms/general.xml

```diff
@@ -0,0 +1,26 @@
+<form>
+    <field>
+        <id>general.heuristics</id>
+        <label>Use Heuristics</label>
+        <type>checkbox</type>
+        <help>Whether to enable or disable the usage or heuristic detection.</help>
+    </field>
+    <field>
+        <id>general.updateperiod</id>
+        <label>Update Period</label>
+        <type>text</type>
+        <help>Time in seconds how ofter to refresh trails.</help>
+    </field>
+    <field>
+        <id>general.adminpassword</id>
+        <label>SHA256 Admin Password</label>
+        <type>text</type>
+        <help><![CDATA[Set the SHA256 password for user admin here. There are many ways to create a password, e.g. from <a href="https://hashgenerator.de/">here</a>]]>. Default password is changeme!</help>
```

Comment:
to prevent transport over web pages, you have three options:

* The user should use the shell: `echo -n changeme! | sha256sum`
* You use a JS implementation like https://code.google.com/archive/p/crypto-js/downloads to encode it directly in the web interface
* you encode on PHP side using hash (https://secure.php.net/manual/de/function.hash.php)

Reply:
I'll add the first message to help text.